### PR TITLE
[MM-54795] Convert `./components/widgets/icons/fa_previous_icon.tsx` to function component

### PR DESCRIPTION
**Pull request by fyfirman**
#### Summary
Convert `./components/widgets/icons/fa_previous_icon.tsx` to function component. I also update the snapshot since it has different result.

#### Ticket Link
Fixes https://github.com/mattermost/mattermost/issues/24759
Jira [MM-54795](https://mattermost.atlassian.net/browse/MM-54795)

#### Release Note

```release-note
NONE
```

[MM-54795]: https://mattermost.atlassian.net/browse/MM-54795?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: webapp/channels/src/components/widgets/icons/fa_previous_icon.tsx [hunk 1]
```diff
@@ -8,21 +8,18 @@ import LocalizedIcon from 'components/localized_icon';
...
-    public render(): JSX.Element {
-        const className = 'icon icon-chevron-left' + (this.props.additionalClassName ? ' ' + this.props.additionalClassName : '');
-        return (
-            <LocalizedIcon
-                className={className}
-                title={{id: t('generic_icons.previous'), defaultMessage: 'Previous Icon'}}
-            />
-        );
-    }
+function PreviousIcon(props: Props) {
```
Comment (larkox) [inline webapp/channels/src/components/widgets/icons/fa_previous_icon.tsx:14]: Exactly the same 3 comments I gave you on https://github.com/mattermost/mattermost/pull/24787

**Review by larkox — CHANGES_REQUESTED** [review]
Great work! It would be great if you could address the same stuff I noted in a different PR. The components are really similar and should be handled in a similar way too :smile:

**Comment by fyfirman** [comment]
I just fixed it @larkox , thanks for the feedback.

**Review by larkox — APPROVED** [review]
Great work. Thanks!

**Review by hmhealey — APPROVED** [review]
Looks great! Thanks for the PR and for helping us move to `defineMessage` as well 🙂

**Comment by hmhealey** [comment]
Skipping QA review since there's not really any functionality to test here

**Pull request merged**